grails-forge-core/src/main/java/org/grails/forge/feature/build/gradle/templates/buildSrcBuildGradle.rocker.raw

```diff
@@ -3,6 +3,7 @@
 @import org.grails.forge.feature.Features
 @import org.grails.forge.build.gradle.GradleBuild
 @import org.grails.forge.build.gradle.GradleDependency
+@import org.grails.forge.util.VersionInfo
```

Comment:
are we still requiring the `buildSrc` folder?

Reply:
That will be addressed on https://github.com/grails/grails-forge/issues/406.